packages/liveblocks-core/src/protocol/Op.ts

```diff
@@ -24,7 +24,7 @@ export type Op =
   | SetParentKeyOp // Only for lists!
   | DeleteObjectKeyOp;
 
-export type CreateOp = CreateRootObjectOp | CreateChildOp;
+export type CreateOp = CreateChildOp;
```

Comment:
Since `CreateChildOp` only existed to exclude the root op type, we can also rename all instances to `CreateOp` now, as all members left have a parent now 👌

Reply:
👍